notebooks/example/anomaly_code.ipynb

```diff
@@ -0,0 +1,809 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    predictor = Predictor(steps=[1, 5], alpha=parameters["predictor"]["sdrc_alpha"])
</pre><p>The parameters defined in this notebook appear to be those selected for the <code>gymdata.csv</code> dataset used in the example <code>hotgym.py</code> is there any work that needs to be done on our part to ensure these parameters are correct for the cpu dataset?</p><p> </p><p>Why are steps 1 and 5  used in the predictor? Why not other values?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/HTM-applications/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>